runtime/executor/src/test/java/org/apache/nemo/runtime/executor/task/TaskExecutorTest.java

```diff
@@ -206,7 +206,7 @@ public void testInvalidInputData() throws Exception {
       // This should not be reached.
       fail();
     } catch (NullPointerException e) {
-      assertEquals(true, true);
+      assert true;
```

Comment:
How about changing this to `assertTrue(true);`  since this is a JUnit test?

Reply:
done